src/libs/Logger.js

```diff
@@ -37,12 +37,21 @@ module.exports = class Logger {
     this.stackLevelIndex = levels.indexOf(stackLevel);
   }
 
+  removeStack (error) {
+    const { message, stack, ...rest } = error;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "stack" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.sagacify.com/coding_rules#rule_key=javascript%3AS1854)